edgedb-client/src/client.rs

```diff
@@ -150,11 +189,66 @@ impl<'a> Sequence<'a> {
         }
     }
 
+    pub fn process_complete(&mut self, cmp: &CommandComplete1)
+        -> Result<(), Error>
+    {
+        if cmp.state_data.len() != 0 {
```

Comment:
I think we should check if `cmp.state_typedesc_id` is `EMPTY_STATE` here?

Reply:
What should we do if state_typedesc_id is non-zero and bytes are zero? ProtocolEncodingError?

Reply:
Yeah, that must be a bug of the server I think.